container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -344,7 +342,7 @@ const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 );
 
 /// We allow for 2000ms of compute with a 6 second average block time.
```

Comment:
```suggestion
/// We allow for 500ms of compute with a 6 second average block time.
```

And is 6 second block time correct?

Reply:
it should be 12